tools/sdk-generation-pipeline/Dockerfile

```diff
@@ -0,0 +1,62 @@
+FROM ubuntu:20.04
```

Comment:
Using images from dockerhub is going to be restricted in a couple months. See https://eng.ms/docs/more/containers-secure-supply-chain/approved-images for details. I don't think this image is used in any sort of live service except for being run as a tool on the pipeline agents? So this may not affect you.

Reply:
We should definitely remove any dependencies on base docker images though. I think you're right to sing out Ben. [Here is the list of MCR Tags](https://mcr.microsoft.com/v2/_catalog)

Reply:
thanks @benbp @scbedd for telling me the restriction. I have changed the base image to `mcr.microsoft.com/oss/mirror/docker.io/library/ubuntu:20.04`